### ISSUE TITLE
Color segmentation for differentiating team members for Football

### ISSUE THREAD

**Issue by ashwinvaswani** [SforAiDl/Playground#104]
Differentiate between teams based on color of jersey. We can even discuss other approaches. What do you think? @Rijul1999

**Comment by Rijul1999** [comment]
Yes I think color of jersey may be the easiest way to handle it. Maybe we could look at this paper https://www.cs.ubc.ca/~murphyk/Papers/cvpr2011-camera.pdf, I'll look for implementations for the same.
